v2/gcs-to-sourcedb/src/test/resources/GCSToSourceDbWithoutReaderIT/events.txt

```diff
@@ -1 +1,5 @@
+{"commitTimestamp":{"seconds":1719846503,"nanos":617919000},"serverTransactionId":"MTY0MDExNjY5NDAxMjEwMDA5Mzg\u003d","recordSequence":"00000000","tableName":"AllDatatypeTransformation","mods":[{"keysJson":"{\"varchar_column\":\"example2\"}","oldValuesJson":"{}","newValuesJson":"{\"bigint_column\":\"1000\",\"binary_column\":\"YmluX2NvbHVtbg\u003d\u003d\",\"bit_column\":\"MQ\u003d\u003d\",\"blob_column\":\"YmxvYl9jb2x1bW4\u003d\",\"bool_column\":true,\"date_column\":\"2024-01-01\",\"datetime_column\":\"2024-01-01T12:34:56Z\",\"decimal_column\":\"99999.99\",\"double_column\":123456.123,\"enum_column\":\"1\",\"float_column\":12345.67,\"int_column\":\"100\",\"text_column\":\"Sample text for entry 2\",\"time_column\":\"143000\",\"timestamp_column\":\"2024-01-01T12:34:56Z\",\"tinyint_column\":\"2\",\"year_column\":\"2024\"}"}],"modType":"UPDATE","numberOfRecordsInTransaction":1,"transactionTag":"","shard":"ls1"}
+{"commitTimestamp":{"seconds":1718781240,"nanos":419055000},"serverTransactionId":"MTQxMTQ5MzExNTc0NTkyNzAzODM\u003d","recordSequence":"00000000","tableName":"AllDatatypeTransformation","mods":[{"keysJson":"{\"varchar_column\":\"example\"}","oldValuesJson":"{}","newValuesJson":"{\"bigint_column\":\"12345\",\"binary_column\":\"U29tZSBiaW5hcnkgZGF0YQ\u003d\u003d\",\"bit_column\":\"U29tZSBiaW5hcnkgZGF0YQ\u003d\u003d\",\"blob_column\":\"U29tZSBiaW5hcnkgZGF0YQ\u003d\u003d\",\"bool_column\":true,\"date_column\":\"2024-01-01\",\"datetime_column\":\"2024-01-01T12:34:56Z\",\"decimal_column\":\"12345.67\",\"double_column\":123.456,\"enum_column\":\"1\",\"float_column\":123.45,\"int_column\":\"123\",\"text_column\":\"Sample text\",\"time_column\":\"143000\",\"timestamp_column\":\"2024-01-01T12:34:56Z\",\"tinyint_column\":\"1\",\"year_column\":\"2024\"}"}],"modType":"INSERT","numberOfRecordsInTransaction":1,"transactionTag":"","shard":"ls1"}
+{"commitTimestamp":{"seconds":1718795517,"nanos":877439000},"serverTransactionId":"MTI4NTc0NTY4OTYwMzkxMDgyNA\u003d\u003d","recordSequence":"00000000","tableName":"AllDatatypeTransformation","mods":[{"keysJson":"{\"varchar_column\":\"example1\"}","oldValuesJson":"{}","newValuesJson":"{\"bigint_column\":\"1000\",\"binary_column\":\"ZXhhbXBsZWJpbmFyeTE\u003d\",\"bit_column\":\"ZXhhbXBsZWJpdDE\u003d\",\"blob_column\":\"ZXhhbXBsZWJsb2Ix\",\"bool_column\":true,\"date_column\":\"2024-01-01\",\"datetime_column\":\"2024-01-01T12:34:56Z\",\"decimal_column\":\"99999.99\",\"double_column\":123456.123,\"enum_column\":\"1\",\"float_column\":12345.67,\"int_column\":\"100\",\"text_column\":\"Sample text for entry 1\",\"time_column\":\"143000\",\"timestamp_column\":\"2024-01-01T12:34:56Z\",\"tinyint_column\":\"1\",\"year_column\":\"2024\"}"}],"modType":"INSERT","numberOfRecordsInTransaction":1,"transactionTag":"","shard":"ls1"}
```

Comment:
In assert conditions, we check for number of rows to be 2. Can you move the "example1" insert to the beginning? So, that the test also tests the filter condition.